restapi/restapi.go

```diff
@@ -120,47 +131,80 @@ func (r *RestAPI) ListUsers() ([]personnel_sync.Person, error) {
 
 	// Iterate through people in list from source to convert to Persons
 	for _, person := range peopleList {
```

Comment:
Since the function is up to 75 lines now, maybe the for loop could be moved to another/new function.  (In that case, the `var sourcePeople ...` line can also be removed.

Reply:
yeah good call